src/components/wallet/view/walletView.js

```diff
@@ -1,20 +1,21 @@
 /* eslint-disable react/jsx-wrap-multilines */
 import React, { Fragment } from 'react';
+import { useSelector } from 'react-redux';
 import { View, Text, ScrollView, RefreshControl } from 'react-native';
 import { useIntl } from 'react-intl';
-
 // Components
 import { Icon } from '../../icon';
 import { MainButton } from '../../mainButton';
 import { CollapsibleCard } from '../../collapsibleCard';
 import { WalletDetails } from '../../walletDetails';
 import { WalletDetailsPlaceHolder } from '../../basicUIElements';
-import { ThemeContainer, WalletContainer } from '../../../containers';
```

Comment:
WalletContainer should not be removed here... only ThemeContainer needs removing.....

![Screenshot 2023-07-11 002815](https://github.com/ecency/ecency-mobile/assets/6298342/24a36942-9238-4cc6-88b8-74120c4d1e77)